.circleci/config.yml

```diff
@@ -1,15 +1,65 @@
 version: 2.1
 orbs:
-  ruby: circleci/ruby@0.1.2 
+  ruby: circleci/ruby@0.1.2
+  python: circleci/python@1.4.0
+  heroku: circleci/heroku@0.0.10
 
 jobs:
   build:
+    working_directory: ~/drink-this-backend
     docker:
       - image: circleci/ruby:2.7.2
+        # environment:
+        #     PGHOST: localhost
+        #     PGUSER: your-app-name
+        #     RAILS_ENV: test
+      - image: postgres:9.5
+        environment:
+          POSTGRES_USER: drink-this-backend
+          POSTGRES_DB: drink-this-backend_test
+          POSTGRES_PASSWORD: drink-this
     executor: ruby/default
     steps:
       - checkout
+      - ruby/bundle-install
+      - run: sudo apt-get install python3-dev
       - run:
           name: Which bundler?
           command: bundle -v
-      - ruby/bundle-install
+  test:
+    executor: ruby/default
+    steps:
+      - checkout
+      - run: sudo apt-get install python3-pip
+      - run: python3 -m pip install numpy
+      - run: python3 -m pip install pandas
+      - run: python3 -m pip install sklearn
+      # - run: 
```

Comment:
Interesting... so `run:` is like inputting the python3 stuff in the CLI?

Reply:
Yes! @markcyen it is basically running those commands on the server for circle ci